### PR TITLE
Ubuntu 16.04

### PR DESCRIPTION
**Pull request by dsundq**
Add support for vas

### CODE REVIEW

File: spec/classes/init_spec.rb [hunk 1]
```diff
@@ -373,15 +373,6 @@
             next
           end
 
-          if check == 'vas' and v[:osfamily] == 'Debian' and v[:release] == '16.04'
```
Comment (ghoneycutt) [inline spec/classes/init_spec.rb]: this should remain, just change 16.04 to 18.04

Comment (ghoneycutt) [inline spec/classes/init_spec.rb]: This should remain, just change 16.04 to 18.04

**Comment by ghoneycutt** [comment]
Hi, thank you for the contribution. We need a test that shows that if you have ubuntu 16.04 and vas 4 that it shows the proper files and and their content through the use of fixtures.

**Comment by boandersson** [comment]
Hi @ghoneycutt, can you please clarify what you mean regarding tests? We have the tests that are included since ubuntu1604 is in the platforms hash - those tests towards the default vas_major_version (4) and are seen in the spec output: "with vas params on Debian with lsbdistrelease 16.04".
Do we need other tests as well?

**Comment by boandersson** [comment]
Hi @ghoneycutt, do you have any clarification on what tests you want us to add?

**Comment by ghoneycutt** [comment]
Thank you everyone! I went ahead and rebased and added a commit to support Puppet 4.9 in PR #153

**Pull request closed without merging**